docs/generated/cmd/tarmak/tarmak_clusters_set-current.rst

```diff
@@ -0,0 +1,41 @@
+.. _tarmak_clusters_set-current:
+
+tarmak clusters set-current
+---------------------------
+
+Set current cluster in config
+
+Synopsis
+~~~~~~~~
+
+
+Set current cluster in config
+
+::
+
+  tarmak clusters set-current [flags]
```

Comment:
I don't know if it is possible with the generated docs, but it would be nice to have also some generated docs for the argument. This is the case now https://docs.tarmak.io/design_cli_ux.html#set-current